RecoEgamma/EgammaTools/src/HGCalShowerShapeHelper.cc

```diff
@@ -3,145 +3,44 @@
 const double HGCalShowerShapeHelper::kLDWaferCellSize_ = 0.698;
 const double HGCalShowerShapeHelper::kHDWaferCellSize_ = 0.465;
 
-HGCalShowerShapeHelper::HGCalShowerShapeHelper(edm::ConsumesCollector &&sumes)
-    : caloGeometryToken_{sumes.esConsumes<CaloGeometry, CaloGeometryRecord>()} {}
 
-void HGCalShowerShapeHelper::initPerEvent(const edm::EventSetup &iSetup, const std::vector<reco::PFRecHit> &pfRecHits) {
```

Comment:
this file is out of sync with the master version in what looks like a missing code format.
Please update in full to avoid having someone dealing with conflicts in possible future backports.